internal/testutil/acc/provider.go

```diff
@@ -38,3 +41,23 @@ func providerAWS() *resource.ExternalProvider {
 		Source:            "hashicorp/aws",
 	}
 }
+
+// ConfigProvider creates a new provider with credentials explicit in config.
+//
+// This can be used when you want credentials different from the default env-vars.
+func ConfigProvider(publicKey, privateKey, baseURL string) string {
```

Comment:
is this func only used by ConfigGovProvider?

Reply:
A good point, changed to private 👍